sauce/features/budget/check-credit-balances/settings.js

```diff
@@ -1,8 +1,29 @@
 module.exports = {
   name: 'CheckCreditBalances',
-  type: 'checkbox',
+  type: 'select',
   default: false,
```

Comment:
The value for `default` should match the value in the `options` section. In this case the value should be `disabled`. It's gets a bit wonky when converting an option from a checkbox to a pull-down but I believe the settings code handles it.

Reply:
I mean to ask about something related to this: Is there a way to handle the new settings if the user previously enabled it with the checkbox? The old on setting should correspond to "highlight".

Reply:
The code that does this mapping is here: https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/source/common/main.js#L40-L48

Unfortunately I don't know that this code is going to work for a setting that goes from checkbox to strings other than '1', '2', '3', etc for values....

Reply:
Good point Josh. Should be easy enough to convert from the string to a number. That also obviates the need for "getting the mode". You can just use the enumRectifyMode like
```
if (this.settings.enabled === this.enumRectifyModes.HIGHLIGHT) {}
```